### ISSUE TITLE
Future Idea: Have multiple inputs so we can OR modules

### ISSUE THREAD

**Issue by alexjiao125** [myjeeed/MakerWear#19]
Can currently AND them together but not OR

**Comment by jonfroehlich** [comment]
Yes, this is a nice idea! We've previously discussed this and believe this could be a really great way of increasing the 'ceiling' for MakerWear (though at a cost of increasing complexity--I think we would want to introduce these modules only to more experienced users or older childre). 

@myjeeed , did you share our Google spreadsheet with Jason and Alex that has ideas like this in it?

**Comment by myjeeed** [comment]
I agree with Jon, we should maintain some consistency for our current set of modules and then we should think of a new consistent way to create the set of inconsistent modules (that are not 1x3 - one input, three output)

Pasting the spreadsheet here as well. We need to add this to our GitHub's readme file. Will open an issue for preparing a readme file and its action items.
https://docs.google.com/spreadsheets/d/1NGcD8RCeaSyZD2Fjg5CHRKstRwcadG_0jy593F6SD4U/edit?usp=sharing

**Comment by jonfroehlich** [comment]
I don't see any of the logic modules in the Google Spreadsheet (e.g., ORs, XORs). Did we ever record these? I agree with Alex (and Majeed and I have discussed this a lot) that you get implicit AND behavior via our architecture, though the end-user is unlikely to think about it that way (fairly abstract thinking).

@myjeeed can you change the green in the spreadsheet to not be so saturated. Hard to read.

**Comment by alexjiao125** [comment]
A simpler option is to have an OR module that would take in two inputs and have 2 outputs.  This would be much more compatible with the current design of the modules.  If you look at the attached picture, the two sensors could connect to the OR module in front of them which would then send inputs to the two LEDs.
![20160606_093126](https://cloud.githubusercontent.com/assets/17015893/15823765/1bee9a3c-2bca-11e6-807b-f724835dcaf4.jpg)

**Comment by myjeeed** [comment]
So a 2x2 module? (2 inputs, 2 outputs)

**Comment by jonfroehlich** [comment]
You could still have 2 inputs and 3 outputs, right? But then, I suppose, it would affect our honeycomb design as we would lose the two blank sides.
